.github/workflows/push-build.yaml

```
@@ -0,0 +1,41 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+jobs:
+  push-image-to-container-registry:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+       # docker/setup-qemu action installs QEMU static binaries, which are used to run builders for architectures other than the host. 
+      - name: set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: all
+
+      - name: log in to container registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_USR }}
+          aws-secret-access-key: ${{ secrets.AWS_PSW }}
+      
+      # creating custom env var
+      - name: set env
+        run: echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - name: build and release
+        env:
+          GIT_PSW: ${{ secrets.GIT_PSW }}
+          BRANCH_NAME: ${{ env.BRANCH_NAME }}
+          AWS_USR: ${{ secrets.AWS_USR }}
+          AWS_PSW: ${{ secrets.AWS_PSW }}
```

```suggestion
          AWS_USR: ${{ secrets.AWS_USR }}
          AWS_PSW: ${{ secrets.AWS_PSW }}
```

@travisn @leseb please confirm if this is correct naming. Thanks

---

They are not set in our action Secret env so they can be whatever you want. I don't have access to these credential. @travisn please add them to list of Secrets env in Rook. Thanks!

---

Ok, I added the two secrets to the github settings!